test/pathforaSpec.js

```diff
@@ -1162,17 +1162,57 @@ describe('API', function () {
     var widget = $('#' + form.id);
     expect(widget.length).toBe(0);
   });
-  
-  if('should not show an impression counter widget without an id', function () {
+
+  it('should not show a widget with hideAfterAction without an id', function () {
+    expect(function() {
+      var form = new window.pathfora.Form({
+        msg: 'subscription',
+        header: 'Header',
+        layout: 'slideout',
+        position: 'bottom-right',
+        displayConditions: {
+          hideAfterAction: {
+            confirm: true
+          }
+        }
+      });
+    }).toThrow(new Error('Widgets with the hideAfterAction displayConditions need a preset id value. Display condition denied.'));
+  });
+
+  it('should show widget with hideAfterAction after no action', function () {
+    var widgetId = 'hideAfterActionWidget1';
     var form = new window.pathfora.Form({
+      id: widgetId,
       msg: 'subscription',
       header: 'Header',
       layout: 'slideout',
       position: 'bottom-right',
       displayConditions: {
-        impressions: {
-          session: 1,
-          total: 5
+        hideAfterAction: {
```

Comment:
i almost wonder if we'll need to support a duration here as well (e.g. i want to hide the widget from anyone that clicked forever but those that have closed for only 7 days). maybe something like:

```
{
     "hideAfterAction": {
          "hide":true,
          "duration": 60 // minutes
     }
}
```